examples/MultiLingual/article.md

```diff
@@ -83,7 +83,7 @@ Here we set the device argument as `"cuda"`. This tells Marqo to use the GPU it
 If you don't have a GPU, remove this argument or set it to `"cpu"`. We encourage using a GPU as it will make the `add_documents` 
 process significantly faster (our testing showed a 6–12x speed up). 
 
-We also set the `auto_refresh` argument to `False`. When indexing large volumes of data we encourage you to set this to False, as it optimises the `add_documents` process. 
+We do not set the `auto_refresh` argument, so it defaults to `False`. When indexing large volumes of data we encourage you to keep this `False``, as it optimises the `add_documents` process. 
```

Comment:
it looks like False has an extra backtick: 
```
`False``
```

Reply:
great catch, thanks